examples/chef/common/chef-operational-state-delegate-impl.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- *    Copyright (c) 2023 Project CHIP Authors
+ *    Copyright (c) 2024 Project CHIP Authors
```

Comment:
nit, sorry, I meant only new files needed this. Old files remain with their old year from my undersanding

Reply:
Right my bad. Done.